### DISCUSSION TITLE
Why is this being done here？

### DISCUSSION THREAD

**Discussion by gjfei** [nuxt-modules/i18n#3135] [General]
### Environment

![image](https://github.com/user-attachments/assets/b2750e16-b470-49e4-bfee-777675f33255)

### Reproduction

![image](https://github.com/user-attachments/assets/41a809b1-3265-4baf-b807-ee8e012fbae7)

### Describe the bug

locales  [ 
 {
    code: 'en-us',
    iso: 'en',
  },
  {
    code: 'vi',
    iso: 'vn',
  },
]
defaultLocale   en

routers   /about
             /home

If the current path is "en-us/about", obtaining the path of "home" through useLocalePath will result in "/home". How to get "en-us/home"

**Comment by BobbieGoede** [comment]
This sounds more like a question so I will convert this to a discussion. Also, without seeing your configuration and function usage there's no way I can understand your question, can you provide either a minimal reproduction or your i18n config and a code snippet that triggers your issue? 

Please format your code snippets like so:
````
```ts
//code here
```
````

**Reply by gjfei** [reply]
![image](https://github.com/user-attachments/assets/f1be1c19-b547-4fa3-98aa-f14e50788688)
![image](https://github.com/user-attachments/assets/cd70bb3f-579a-46ad-897d-f227150f65e1)

this is my code

If the current url is "/zh-cn/about" I want to use localePath('/home') to get "/zh-cn/ home"
